documentation/staging/content/userguide/managing-domains/domain-events.md

```diff
@@ -85,7 +88,7 @@ $ kubectl get events -n [namespace] --selector=weblogic.createdByOperator=true
 To get all the events that are generated by the operator for a particular domain resource, for example `sample-domain1`, run:
 
 ```shell
-$ kubectl get events -n [namespace] --selector=weblogic.domainUID=sample-domain1,weblogic.createdByOperator=true --sort-by=lastTimestamp
+$ kubectl get events -n [namespace] --selector=weblogic.domainUID=sample-domain1&&weblogic.createdByOperator=true --sort-by=lastTimestamp
```

Comment:
Does "," not work in all cases? We have a customer with an older version of k8s that can not use '&&'.

Reply:
Hmm, I thought this has been fixed/changed. This could be a merge issue; the old code came back when I merged my branch with the fix you made in main branch.

Reply:
fixed